geode-core/src/main/java/org/apache/geode/internal/DataSerializableFixedID.java

```diff
@@ -823,6 +823,7 @@
   short GATEWAY_SENDER_QUEUE_ENTRY_SYNCHRONIZATION_MESSAGE = 2181;
   short GATEWAY_SENDER_QUEUE_ENTRY_SYNCHRONIZATION_ENTRY = 2182;
   short ABORT_BACKUP_REQUEST = 2183;
+  short EXPIRE_CLIENT_TRANSACTIONS = 2184;
```

Comment:
consider using the old message instead of adding a new one

Reply:
Replace the existing TXRemovalMessage with this new message.